src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ActiveConfiguredProjectsLoader.cs

```diff
@@ -57,43 +55,14 @@ protected override void Dispose(bool disposing)
 
         private async Task OnActiveConfigurationsChangedAsync(IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>> e)
         {
-            await GetLoadedProjectsAsync(e);
-        }
-
-        protected override IDisposable? LinkExternalInput(ITargetBlock<IProjectVersionedValue<IEnumerable<ConfiguredProject>>> targetBlock)
-        {
-            IReceivableSourceBlock<IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>>> sourceBlock = _activeConfigurationGroupService.ActiveConfigurationGroupSource.SourceBlock;
```

Comment:
it need JoinUpstream to the ActiveConfigurationGroupSource

Reply:
done.

ActiveConfigurationGroupSource added to JoinUpstream